mod/groups/views/default/groups/invitationrequests.php

```diff
@@ -46,6 +63,14 @@
 		}
 	}
 	echo '</ul>';
+
+	if (!empty($count)) {
+		echo elgg_view('navigation/pagination', array(
```

Comment:
I know. I was working on the entity views and came across this overblown limit=>0